src/util/loanSearch/dataUtils.js

```diff
@@ -53,6 +54,37 @@ export async function runLoansQuery(apollo, loanSearchState, origin) {
 	return { loans: flssData?.values?.filter(loan => loan !== null) ?? [], totalCount: flssData?.totalCount ?? 0 };
 }
 
+/**
+ * Runs the query to get recommended loans
+ *
+ * @param {Object} apollo The Apollo client instance
+ * @param {Object} options Query options including userId and origin
+ * @param {Number} options.userId The user ID to get recommendations for (optional)
+ * @param {String} options.origin Origin of query formatted as web:##page-context##
+ * @param {Object} options.filterObject Optional filters to apply to recommendations
+ * @param {String} options.sortBy Sort option for recommendations (defaults to 'personalized')
+ * @param {Number} options.limit Limit the number of recommendations (optional)
+ * @returns {Object} The results of the recommendations query
+ */
+export async function runRecommendationsQuery(apollo, {
```

Comment:
This new method could use some unit tests as well.

Reply:
done